packages/ember-routing/lib/system/router.js

```diff
@@ -1563,4 +1564,9 @@ function representEmptyRoute(liveRoutes, defaultParentState, route) {
   }
 }
 
+deprecateProperty(EmberRouter.prototype, 'router', 'routerMicrolib', {
+  id: 'ember-router.router',
+  until: '2.16'
```

Comment:
qq: how do i derive the URL from the section I added to the website? I see that there's a URL with a hash pointing to `http://emberjs.com/deprecations/v2.x/#toc_*`, but I'm not sure how it generates that URL. 

I've been trying to run bundler for the website, but keep getting errors installing deps...